website/docs/plugin/testing/migrating.mdx

```diff
@@ -16,8 +16,18 @@ This guide provides information and examples for most common use cases, but it d
 Take the following steps when you migrate a provider's acceptance tests from SDKv2 to the testing module:
 
 - Change all instances of the following Go import statements in `*_test.go` files:
-    - github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest to github.com/hashicorp/terraform-plugin-testing/helper/acctest
-    - github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource to github.com/hashicorp/terraform-plugin-testing/helper/resource
-- Run `go get github.com/hashicorp/terraform-plugin-testing@v1.0.0`
-- Run `go mod tidy`
-- Verify everything is working as expected.
+    - `github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest` to `github.com/hashicorp/terraform-plugin-testing/helper/acctest`
+    - `github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource` to `github.com/hashicorp/terraform-plugin-testing/helper/resource`
+- Get and download terraform-plugin-testing `v1.0.0`:
```

Comment:
Perhaps suggest using `latest` rather than specific version?

Reply:
That's a good point, although pointing to `v1.0.0` ensures that there would be no breaking changes in the migration. Maybe we can add an additional optional step for those who want to upgrade to the latest version after making sure that the migration to `v1.0.0` works as expected?

Reply:
Theoretically `github.com/hashicorp/terraform-plugin-testing@latest` should always be safe because:

- Go's import system uses separate import paths for new major versions, so v2 would be `github.com/hashicorp/terraform-plugin-testing/v2@...`
- All versions of v1 should have compatibility promises for existing functionality